medical/models/medical_physician.py

```diff
@@ -98,10 +98,3 @@ class MedicalPhysician(models.Model):
     )
 
     _defaults = {'is_doctor': True, 'supplier': True, 'active': True, }
-
-    @api.model
-    def create(self, vals,):
-        groups_proxy = self.env['res.groups']
-        group_ids = groups_proxy.search([('name', '=', 'Medical Doctor')])
-        vals['groups_id'] = [(6, 0, group_ids)]
```

Comment:
Mind if we push this off? We're in the process of breaking `medical_physician` into it's own module, and this change has already been made there. We added a bunch of testing as well (this medical_physician doesn't really have much in the way of tests)